src/migration-builder.ts

```diff
@@ -329,13 +331,16 @@ export default class MigrationBuilderImpl implements MigrationBuilder {
     this.func = PgLiteral.create
 
     // expose DB so we can access database within transaction
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const wrapDB = <T extends any[], R>(operation: (...args: T) => R) => (...args: T) => {
-      if (this._REVERSE_MODE) {
-        throw new Error('Impossible to automatically infer down migration')
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    const wrapDB =
+      <T extends any[], R>(operation: (...args: T) => R) =>
+      (...args: T) => {
+        if (this._REVERSE_MODE) {
+          throw new Error('Impossible to automatically infer down migration')
+        }
+        return operation(...args)
       }
-      return operation(...args)
-    }
+    /* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
This is just a workaround for now, cause when I move line `eslint-disable-next-line` just one line down, the formatting breaks completely
This part is already ugly ^^ IMO we could refactor it to get rid of this inline generic function later in another PR or so

Reply:
or we should use some alternative to those strange (to my taste) prettier formatting rules :D

Reply:
I think the only rule change that could help here would be increment the `printWidth`. But this is already on `120` and this should not increased to a higher value.
So what alternative did you think of? **Not** using prettier would **not** be an option to me, cause I use it in all my projects, it has a wide eco-system and it's the best formatting tool out there with a strong team behind.